src/main/resources/assets/computercraft/lua/rom/programs/http/wget.lua

```diff
@@ -15,7 +15,12 @@ if not http then
     printError( "Set http_enable to true in ComputerCraft.cfg" )
     return
 end
- 
+
+local function getFilename( sUrl )
+    sUrl = sUrl:gsub( "[#?].*" , "" ):gsub( "/+$" , "" )
+    return sUrl:match( "/([^/]+)$" ) or " "
```

Comment:
I'm not sure if the `or " "` should be here or further down(See comment on line 52)